core/src/datayoga_core/blocks/redis/lookup/block.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "redis.write",
+  "description": "Write to a Redis data structure",
+  "type": "object",
+  "properties": {
+    "connection": { "title": "Connection name", "type": "string" },
+    "cmd": {
+      "title": "Redis command",
+      "description": "The command to execute",
+      "type": "string"
+    },
+    "args": {
+      "title": "Redis command arguments",
+      "description": "The list of expressions produces arguments",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "language": {
+      "description": "Language",
+      "type": "string",
+      "enum": ["jmespath", "sql"]
+    },
+    "field": {
+      "type": "string",
+      "title": "Target field",
+      "description": "The field to write the result to"
+    },
+    "reject_on_error": {
+      "type": "boolean",
+      "default": false,
+      "title": "Reject on error",
+      "description": "Should reject a record on error or not in case if the key is of a different type"
```

Comment:
If the key is of a different type, there's no point not to reject it, or it would be stuck, no? I'd consider removing this flag completely.

Reply:
✅